apps/tenant-management-webapp/src/app/pages/admin/services/calendar/events/eventList.tsx

```diff
@@ -209,6 +210,8 @@ export const EventList = ({ calendarName }: EventListProps): JSX.Element => {
   // eslint-disable-next-line
   useEffect(() => {}, [selectedEvents]);
 
+  console.log(next);
```

Comment:
remove console.log

Reply:
Thanks, @thdlam. The issue shall be fixed!